codegen-core/src/test/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/BuilderGeneratorTest.kt

```diff
@@ -140,4 +144,29 @@ internal class BuilderGeneratorTest {
         }
         project.compileAndTest()
     }
+
+    @Test
+    fun `don't add serde to error types`() {
+        val provider = testSymbolProvider(model)
+        val project = TestWorkspace.testProject(provider)
+        project.moduleFor(errorStruct) {
+            rust("##![allow(deprecated)]")
+            StructureGenerator(model, provider, this, errorStruct, emptyList()).render()
+            implBlock(provider.toSymbol(errorStruct)) {
+                BuilderGenerator.renderConvenienceMethod(this, provider, errorStruct)
+            }
+        }
+        project.withModule(provider.moduleForBuilder(errorStruct)) {
+            BuilderGenerator(model, provider, errorStruct, emptyList()).render(this)
+        }
+        project.compileAndTest()
```

Comment:
Doesn't this test need to also enable the cfg flag and the serde features?

Reply:
I thought I enabled unstable feature by default, but it seems like I didn't. 

Let me fix it up.

Reply:
Yes, you are right. 

I initially made the unstable flags to be enabled by default, and I forgot to change it.

Thanks for pointing it out.

Reply:
No, this ones is meant to test the generated code.
It only generates code for error struct, and it checks if it has any `serde::` attached to it.

So, I think you don't really need the `serde` features enabled.